src/token.ts

```diff
@@ -16,7 +16,8 @@ function encode64(data: any) {
  * @returns The signature of the token in base64.
  */
 function sign(apiKey: string, encodedHeader: string, encodedPayload: string) {
-  return createHmac('sha256', apiKey)
+  return crypto
+    .createHmac('sha256', apiKey)
```

Comment:
It could be also something like this 🤔 

```js
async function sign(apiKey: string, encodedHeader: string, encodedPayload: string) {
  const { createHmac } = await import('crypto')
  return createHmac('sha256', apiKey)
    .update(`${encodedHeader}.${encodedPayload}`)
    .digest('base64')
    .replace(/\+/g, '-')
    .replace(/\//g, '_')
    .replace(/=/g, '')
}
```

This can also provide useful code splitting or better compatibility in environments where the crypto module might not be available. In Node.js, once a module is loaded, it should be cached. So, subsequent imports of the same module don't actually hit the disk, they just retrieve the module from the cache.

I also didn't test, just throwing it out 😅

Reply:
I will try it, thanks!